ui/src/components/app/footerLinks/Impressum.js

```diff
@@ -1,13 +1,19 @@
 import React, { Component } from "react";
 
 class Impressum extends Component {
+
+    componentDidMount(){
+      document.title = "Impressum | o2r Demoserver"
```

Comment:
This applies here and multiple other places: We want `"| o2r Demoserver"` to be defined only _once_. Imagine you installing this on your own server and you want to change the name, this requires you to go into multiple files, which makes it harder to keep your codebase up to date etc.

This should go into the configuration file at https://github.com/o2r-project/o2r-UI/blob/master/ui/src/helpers/config.json

Reply:
The String " | o2r Demoserver" is now defined in the config JSON (ui/src/helpers/config.json) and is used for the document titles.

https://github.com/Fmazin/o2r-UI/blob/03680d383ffa68ed31cb9c8d7c548982ba31dc85/ui/src/helpers/config.json#L4

It is used in this way: 
```
    componentDidMount() {
        this.getCompendia();
        document.title = "Author View" + config.title;
    }
```